src/main.ts

```diff
@@ -52,13 +52,24 @@ function main() {
   // BLOG_URLには自分のサイトのURLを入力しておくこと。
   // 例: https://budougumi0617.github.io/
   let bookmarks = -1;
-  const url = PropertiesService.getScriptProperties().getProperty("BLOG_URL");
-  if (url != null) {
-    bookmarks = getBookmarcCount(url);
+  const blogUrl = PropertiesService.getScriptProperties().getProperty("BLOG_URL");
+  if (blogUrl != null) {
+    bookmarks = getBookmarkCount(blogUrl);
   } else {
     console.log("はてなブックーマーク数は取得しませんでした");
   }
 
-  console.log([today, followers, pv, bounceRate, bookmarks]);
-  sheet.appendRow([today, followers, pv, bounceRate, bookmarks]);
+  // 「指定した url のブログ」に対する読者数を取得する
+  // BLOG_URLには自分のサイトのURLを入力しておくこと。
+  // 例: https://budougumi0617.github.io/
+  let numOfSubscribers = -1;
+  const hatenaBlog = PropertiesService.getScriptProperties().getProperty("HATENA_BLOG");
+  if (hatenaBlog === "true") {
```

Comment:
```suggestion
  if (hatenaBlog === "true" && blogUrl != null) {
```
ってしないと、`hatenaBlog`だけプロパティがあったとき`null`が返ってました（まあエラーで落ちたりはしないのでいいと言えばいいのですが）